projects/aca-content/src/lib/components/view-profile/view-profile.component.html

```diff
@@ -1,10 +1,9 @@
-<div class="app-profile-container">
+<div class="app-profile-container" [ngClass]="{'app-view-profile-container' : hideSidenav === true}">
   <div class="app-profile-row">
     <div class="app-profile-title">
       <mat-icon class="app-profile-icon" (click)="navigateToPersonalFiles()" id="backButton">arrow_back</mat-icon>
```

Comment:
should be navigating to the default path driven by the configuration, we already had this support before